lte/gateway/deploy/agw_install_docker.sh

```diff
@@ -11,6 +11,7 @@
 # limitations under the License.
 
 MODE=$1
+RERUN=0    # Set to 1 to skip network configuration and run ansible playbook only
```

Comment:
Not for this PR, but a future PR could consider something like Ansible's idempotency rules to see if network configuration needs to be done at all based on the state of the host.